arm-sql/2014-04-01/swagger/auditingPolicies.json

```diff
@@ -336,7 +336,7 @@
       "in": "path",
       "required": true,
       "type": "string",
-      "description": "The subscription credentials which uniquely identify Microsoft Azure subscription. The subscription ID forms part of the URI for every service call."
+      "description": "The subscription credentials which uniquely identify a Microsoft Azure subscription. The subscription ID forms part of the URI for every service call."
```

Comment:
I think this is the one you came up with last week :P  I'll change it to something simple.

---
In reply to: [85462943](https://github.com/Azure/azure-rest-api-specs/pull/555#discussion_r85462943) [](ancestors = 85462943)